src/viur/datastore/cache.py

```diff
@@ -0,0 +1,57 @@
+import sys
+from typing import Any, Dict, List, Union
+
+from viur.datastore.config import conf
+from viur.datastore.types import Entity, Key
+
+memcache_max_batch_size = 30
+memcache_namespace = "viur-datastore"
+memcache_timeout = 60 * 60
+memcache_max_size = 1_000_000
```

Comment:
They look like constants. So I would write them in UPPER_SNAKE_CASE.

```suggestion
MEMCACHE_MAX_BATCH_SIZE = 30
MEMCACHE_NAMESPACE = "viur-datastore"
MEMCACHE_TIMEOUT = 60 * 60
MEMCACHE_MAX_SIZE = 1_000_000
```